src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -144,6 +146,20 @@ void PeakDetectorCLI::processOptions(int argc, char* argv[]) {
 			uploadToPolly_bool = true;
 			}
 			break;
+		case 'N':
+			//parse polly project argument here.
+			{
+			pollyProject = QString(optarg);
+			pollyProject_provided = true;
+			}
+			break;
+		case 'S':
+			//Upload sample cohort file from the CLI to polly.
+			{
+			sample_cohort_file = QString(optarg);
+			sample_cohort_present  =true;
```

Comment:
@Ky5595 Why not check for empty string instead of using a global boolean variable? Is there some corner case here?

Same for case 'P' and 'N' too.